goldie.go

```diff
@@ -78,17 +78,17 @@ func Assert(t *testing.T, name string, actualData []byte) {
 // can be explicitly called if needed. The more common approach would be to
 // update using `go test -update ./...`.
 func Update(name string, actualData []byte) error {
-	err := ensureFixtureDir()
+	err := ensureDir(FixtureDir)
 	if err != nil {
 		return err
 	}
 
-	err = ioutil.WriteFile(goldenFileName(name), actualData, FilePerms)
-	if err != nil {
-		return err
-	}
+	// if err = ensureDir(goldenFileName(name)); err != nil {
```

Comment:
Please remove these commented lines :bowing_man:

Reply:
Oh, yeah I flubbed a lot on this. Lines 86-88 were the lines to create the directories for other stuff. Since they're commented out, I've effectively made no changes.

Uncommenting those lines also causes a lot of other problems - I'll have to double back and walk through a lot of stuff on this. I'll bump the PR when I've done that.

Reply:
But it's redundant now, right? If so, we could just remove these lines.

Reply:
The commented lines were the new bit, and had a problem that I had to correct.

The uncommented lines (they're now gone) were the old stuff that just ensured FixtureDir. Like I said, those are now gone.